src/applications/disability-benefits/all-claims/pages/incidentDate.js

```diff
@@ -2,15 +2,15 @@ import currentOrPastDateUI from 'us-forms-system/lib/js/definitions/currentOrPas
 
 import { ptsd781NameTitle } from '../content/ptsdClassification';
 import { ptsdDateDescription } from '../content/incidentDate';
-import fullSchema from '../config/schema';
+import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
 
-const { date } = fullSchema.definitions.ptsdIncident.properties;
+const { incidentDate } = fullSchema.definitions.ptsdIncident.properties;
```

Comment:
Heh, I think they just changed this to be `date` in one of the last PRs. :man_shrugging:  (Sure, _now_ I tell you. :roll_eyes: :stuck_out_tongue: )

Reply:
I did that. And now that I have some more context I'm undoing it  😄